stylesheets/global-style.css

```diff
@@ -0,0 +1,374 @@
+@charset "UTF-8";
+
+body {
+  text-align: center;
+  color: black;
+	margin-top: 0px;
+	margin-right: 0px;
+	margin-bottom: 0px;
+	margin-left: 0px;
+	font-style: normal;
+	font-weight: 200;
+}
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0;
+  display: block;
+  margin: 0 auto;
+  max-width: 100%;
+}
+
+html, body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+/* Header */
+header {
+  background-color: #2952a3;
+  color: #99CC33;
+}
+
+.total {
+  color: white;
+}
+
+.header-container {
+  width: 100%;
+  height: 100px;
+  padding-bottom: 50px;
+  background-color: #193366;
+  float: left;
+  clear: none;
+  margin: 0;
+}
+
+.header-container-dogs {
+  width: 100%;
+  height: 100px;
+  padding-bottom: 50px;
+  background-color: #193366;
+  margin: 0;
+}
+
+.header-container-blog {
+  width: 100%;
+  height: 100px;
+  padding-bottom: 50px;
+  background-color: #193366;
+  margin: 0;
+}
+
+.header-container-checkout {
+  width: 100%;
+  height: 100px;
+  padding-bottom: 50px;
+  background-color: #193366;
+  margin: 0;
+}
+
+/* Navbar */
+.navbar-container {
+  background-color: #C0C0C0;
+  width: 100%;
+  height: auto;
+  padding-bottom: 15px;
+  padding-top: 2px;
+  float: left;
+  clear: none;
+}
+
+.navbar-container a {
+  color: #193366;
+  font-weight: normal;
+}
+
+.hero-container {
+  width: 100%;
+  height: auto;
+  padding-bottom: 50px;
+  padding-top: 50px;
+  float: left;
+  clear: none;
+}
+
+.hero {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.hero p {
+  margin-right: 100px;
+}
+
+.logo {
+  font-weight: bold;
+	float: left;
+	margin-top: 15px;
+	margin-left: 25px;
+}
+
+.checkout {
+  display: inline-flex;
+  flex-direction: column-reverse;
+  flex-flow: column wrap;
+  float: right;
+  padding: 10px;
+}
+
+.button {
+  font: bold 11px Arial;
+  text-decoration: none;
+  background-color: #99CC33;
+  color: #333333;
+  border: 1px solid #CCCCCC;
+}
+
+.see-dogs {
+  font: bold 20px Times;
+  text-decoration: none;
+  background-color: #D0D0D0;
+  color: #2952a3;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.submit-button {
+  font: bold 20px Times;
+  text-decoration: none;
+  background-color: #99CC33;
+  color: white;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.navbar-container a {
+  font-family: 'Merriweather Sans', sans-serif;
+  font-size: 30px;
+}
+
+nav ul {
+  list-style-type: none;
+}
+
+nav ul li {
+	float: left;
+	color: #FFFFFF;
+	margin-right: 2%;
+	letter-spacing: 2px;
+}
+ul li a {
+  text-decoration: none;
+}
+
+.dog-container {
```

Comment:
On the Dog Page the grid should be adjusted to 4 columns by default to match the updated design.